Example in README does not work

When I try to run the example from the README, I get 
```
$ go run main.go
panic: open dicom.dic: no such file or directory

goroutine 1 [running]:
github.com/gillesdemey/go-dicom.NewParser(0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/.../go/src/github.com/gillesdemey/go-dicom/parser.go:45 +0xf9
main.main()
	/.../go/src/github.com/micahhausler/dicom/main.go:38 +0x2a9
exit status 2
```

--- a/data.go
+++ b/data.go
@@ -1,4 +1,6 @@
-#
+package dicom
+
+const dicomDictData = `#
 #  Copyright (C) 1994-2012, OFFIS e.V.
 #  All rights reserved.  See COPYRIGHT file for details.
 #
@@ -3949,3 +3951,4 @@
 #
 # EOF
 #
+`

--- a/dictionary.go
+++ b/dictionary.go
@@ -1,6 +1,7 @@
 package dicom
 
 import (
+	"bytes"
 	"encoding/csv"
 	"io"
 	"strconv"
@@ -15,10 +16,11 @@ type dictEntry struct {
 }
 
 // Sets the dictionary for the Parser
-func Dictionary(r io.Reader) func(*Parser) error {
+func Dictionary(b []byte) func(*Parser) error {
 
 	return func(p *Parser) error {
 
+		r := bytes.NewReader(b)
 		reader := csv.NewReader(r)
 		reader.Comma = '\t'  // tab separated file
 		reader.Comment = '#' // comments start with #

--- a/parser.go
+++ b/parser.go
@@ -2,7 +2,6 @@ package dicom
 
 import (
 	"fmt"
-	"os"
 	"strings"
 )
 
@@ -38,14 +37,7 @@ func NewParser(options ...func(*Parser) error) (*Parser, error) {
 	p := Parser{}
 
 	// apply defaults
-	fh, err := os.Open("dicom.dic")
-	defer fh.Close()
-
-	if err != nil {
-		panic(err)
-	}
-
-	err = Dictionary(fh)(&p)
+	err := Dictionary([]byte(dicomDictData))(&p)
 
 	if err != nil {
 		panic(err)

--- a/parser_test.go
+++ b/parser_test.go
@@ -1,7 +1,6 @@
 package dicom
 
 import (
-	"os"
 	"testing"
 )
 
@@ -18,14 +17,7 @@ func TestConstructor(t *testing.T) {
 }
 
 func TestDictionaryOption(t *testing.T) {
-
-	fh, err := os.Open("dicom.dic")
-	defer fh.Close()
-	if err != nil {
-		t.Error(err)
-	}
-
-	p, err := NewParser(Dictionary(fh))
+	p, err := NewParser(Dictionary([]byte(dicomDictData)))
 	if err != nil {
 		t.Error(err)
 	}

